File: Pascal/1005/1005.pas
var
	A, B, MEDIA : double;
begin
	readln(A);
	readln(B);

	MEDIA := ((A*3.5) + (B*7.5)) / (3.5 + 7.5);

	writeln('MEDIA = ', MEDIA:5:5);
end.

File: C++/1005/1005.cpp
#include <iostream>
using namespace std;

double A, B, MEDIA;

int main() {
	scanf("%lf", &A);
	scanf("%lf", &B);

	MEDIA = ((A*3.5) + (B*7.5)) / (3.5 + 7.5);

	printf("MEDIA = %.5f\n", MEDIA);
}
